_sass/components/book.scss

```diff
@@ -79,6 +79,12 @@
 		margin-bottom: 2em;
 	}
 
+	ol ul,
+	ul ul {
+		margin-top: 1em;
+		margin-bottom: 1em;
```

Comment:
I think you could probably go smaller, to `0.5rem`. But if you do that, you'll also want to change `li + li` below to match, so each bullet point has an equal distance from the bullet point above/below it.